### ISSUE TITLE
User input mod-N counter

### ISSUE THREAD

**Issue by surajkadapa** [surajkadapa/3_bit_up#4]
Take the user input for the required N(<16) and make the counter count up to that and reset to it's initial state, no hardcoding of values allowed.

**Comment by kavs2803** [comment]
can i be assigned this?

**Comment by surajkadapa** [comment]
@kavs2803 Yes. The time limit is 1 hour. If not, others can ask to be assigned.

**Comment by euphoricair7** [comment]
Could You assign me this issue?

**Comment by ankitkumar099** [comment]
can you assign this issue??

**Comment by ananyashreyaz** [comment]
can I get assigned?

**Comment by surajkadapa** [comment]
@euphoricair7 Yes. The time limit is 1 hour. If not, others can ask to be assigned.

**Comment by IshaanNene** [comment]
Can i be assignedthis issues

**Comment by surajkadapa** [comment]
@euphoricair7 reassigning the issue since out of time.

**Comment by surajkadapa** [comment]
@IshaanNene Yes. The time limit is one hour.

**Comment by SnehaBangera** [comment]
can i be assigned

**Comment by 411sst** [comment]
can i get assigned if this is still open?

**Comment by surajkadapa** [comment]
@SnehaBangera Yes. The time limit is 1 hour. If not, others can ask to be assigned.

**Comment by surajkadapa** [comment]
@SnehaBangera Yes. The time limit is 1 hour. If not, others can ask to be assigned.

**Comment by Alley2717** [comment]
can i try this

**Comment by surajkadapa** [comment]
@SnehaBangera any update?
